core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -150,6 +150,7 @@ message DeletePOptions {
   optional bool alluxioOnly = 2;
   optional bool unchecked = 3;
   optional FileSystemMasterCommonPOptions commonOptions = 4;
+  optional bool resetDirectChildrenLoadedState = 5;
```

Comment:
ok now you passed this to the master by this RPC, how is the master using this field?

Reply:
Sorry, I push a part of changes accidentally.

Reply:
will "setDirectChildrenUnloaded" be a better naming? @jiacheliu3